packages/ui/src/components/SliderSteps/SliderStep.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Circle } from '@chakra-ui/react';
+
+export interface SliderStepProps {
+  current: boolean;
+  onClick: () => void;
+}
+const currentDotProps = {
+  backgroundColor: 'brand.500',
+  mx: '2px',
```

Comment:
🤔 Shouldn't it be 6px? I don't know if this was supposed to be the effect, but by clicking on the dots on the storybook, I noticed that the margins jump strangely when switching to active.